phpcs.xml.dist

```diff
@@ -88,7 +88,8 @@
 	</rule>
 
 	<rule ref="WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid">
-		<exclude-pattern>tests/phpunit/integration/includes/REST_Setup.php</exclude-pattern>
+		<exclude-pattern
+    >tests/phpunit/integration/includes/REST_Setup.php</exclude-pattern>
```

Comment:
?

Reply:
This got added in a merge conflict, will update.